gatsby-config.ts

```diff
@@ -14,6 +15,9 @@ require(`dotenv`).config({
 });
 
 const config: GatsbyConfig = {
+  flags: {
```

Comment:
Not sure if it hurts anything to have this, however I like it as it flags build errors during development